app/boards/intel_adsp_cavs25.conf

```diff
@@ -30,6 +30,7 @@ CONFIG_COMP_VOLUME_WINDOWS_FADE=y
 CONFIG_COMP_DRC=y
```

Comment:
@singalsu checkpatch does not like commits without description. as other checks are green, I think I'll let this pass through, but note for future.